lib/services/local_file_update_service.dart

```diff
@@ -257,9 +257,7 @@ class LocalFileUpdateService {
 
       for (ente.File file in enteFiles) {
         final Location? location = await tryLocationFromExif(file);
-        if (location != null &&
-            (location.latitude ?? 0) != 0.0 &&
-            (location.longitude ?? 0) != 0.0) {
+        if (location != null && Location.isValidLocation(location)) {
```

Comment:
Can remove the `location != null` check as `Location.isValidLocation()` checks that and returns false.

Reply:
So added this because idea was crying while we were using it in the bottom.